src/cljs/main/broadfcui/common/flex_utils.cljs

```diff
@@ -4,12 +4,12 @@
     ))
 
 
-(defn flex-box [attrs & children]
+(defn box [attrs & children]
```

Comment:
I once had Big Plans for these utilities.  Modeled after a combination of Java Swing's BoxLayout and Apple's Interface Builder.  I think there's other places that could be replaced by `flex/box`es.